### ISSUE TITLE
how to set kernel_size 

### ISSUE THREAD

**Issue by liwenssss** [BaowenZ/RaDe-GS#39]
Hi, thanks for your excellent work! I noticed that in your rendering process, you add a kernel_size parameter. However, I couldn't find any issue regarding its settings. It seems that the kernel_size is set to 0. Could you please explain the purpose of this parameter and how I should adjust it based on experimental results?

**Comment by BaowenZ** [comment]
Hi, it is 2D filter size. Since we did not use it in our experiments, we have set it to 0. You could use it to address aliasing caused by low resolution or small 2D kernel size.

**Comment by liwenssss** [comment]
Thanks for your reply. Another question, I want to save the rendered normal map. I first normalize the rendered_normal to the [0, 1] range and then use torchvision.utils.save_image to save it. However, the saved result appears yellow or even red. Is this normal?

**Comment by BaowenZ** [comment]
Hi, we show the normal by:
```
plt.imshow((-render_pkg["normal"].detach().cpu().numpy().transpose(1,2,0)+1)/2)
```

**Comment by liwenssss** [comment]
Hi, I have another question. I want to use your module to get normal vector, But I dont know how to do. Is it any possible to achieve it by your module?
